src/lib/shSdkWallet.ts

```diff
@@ -20,7 +20,11 @@ type ISpendOptions = Omit<Parameters<typeof spend>[2], 'onAccount' | 'onNode'>
     payload?: Encoded.Any,
   }
 
-export class ShSdkWallet extends AeSdkWallet {
+/**
+ * Class extends `AeSdkWallet` from aepp-sdk-js
+ * provides flexibility to manage the accounts the way wallet would like to handle
+ */
+export class ShAeSdk extends AeSdkWallet {
```

Comment:
1. I suggest to use `AeShSdkWallet` or `ShAeSdkWallet` here to maintain the base class naming strategy.
2. Good practice is to name the file the same way as the exported class name.

Reply:
Renamed it to `AeSdkSupehero` to maintain the base class naming strategy.

AeSdkBase -> AeSdkWallet -> AeSdkSuperhero